src/src/com/microsoft/aad/adal/AuthenticationSettings.java

```diff
@@ -158,24 +162,38 @@ public void setActivityPackageName(String activityPackageName) {
     }
 
     /**
-     * Skip broker usage.
-     * 
-     * @return true if broker is not used.
+     * @deprecated As of release 1.1.14, replaced by {@link #getUseBroker()}
      */
-    public boolean getSkipBroker() {
+    @Deprecated public boolean getSkipBroker() {
         return mSkipBroker;
     }
 
     /**
-     * Sets flag to skip broker.
-     * 
-     * @param skip True to not use broker
+     * @deprecated As of release 1.1.14, replaced by {@link #setUseBroker()}
      */
-    public void setSkipBroker(boolean skip) {
+    @Deprecated public void setSkipBroker(boolean skip) {
```

Comment:
Do you think it will be good that we just use useBroker in setSkipBroker and getSkipBroker instead of @Deprecated?